xds/src/test/certs/badclient.pem

```diff
@@ -0,0 +1,17 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
Can we reuse https://github.com/grpc/grpc-java/tree/master/testing/src/main/resources/certs directly?

Reply:
That's what I had before. @creamsoup  didn't want to depend on external files/resources and I tend to agree to minimize chances of breakage

Reply:
i am a bit against using other module's file. 
we could do that if we change the interface to accept input stream and modify the build to properly copy the resources.

Reply:
It's just adding a test scope dependency of grpc-testing library and calling `getResourceAsStream()`. 
```
testCompile project('grpc-testing')
```

The grpc-testing library (available in Maven `io.grpc:grpc-testing`) is designed for testing purpose, even for external users. I don't think there is problem for sds test.

Reply:
Also I would like to be able to change the files without breaking others. As I understand these files under "resources" are used by examples so let's not depend on them.

Reply:
If you need a custom cert file then you can not modify and use the resources from grpc-testing, otherwise it may break the users.

Reply:
@dapengzhang0 yes, we are saying the same thing. That is one reason I don't want to use the files from grpc-testing.